### PR TITLE
Adding Travis CI

### PR DESCRIPTION
**Pull request by wogsland**
To start Travis CI automation after accepting pull request,
1). Login to https://travis-ci.org with the borisrepl github account.
2). Click + next to "My Repositories" on the left.
3). Turn on borisrepl/boris.
4). Push to master and Travis will build and run all the unit tests on several current versions of PHP.

(Also removed unused credential file so automated unit testing would work.)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: php
+php:
+  - "5.6"
```
Comment (GrahamCampbell) [inline .travis.yml]: :-1:

Comment (GrahamCampbell) [inline .travis.yml]: Should be without the quotes.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  - "5.5"
+  - "5.4"
+  - "5.3"
+  - "hhvm"
+script: phpunit --bootstrap tests/autoload.php -c tests.xml
+notifications:
```
Comment (GrahamCampbell) [inline .travis.yml]: :-1: This is the default so should be omitted.

**Comment by wogsland** [comment]
@GrahamCampbell Thanks for pointing those out - haven't looked at the Travis documentation in a while. If it ain't broke...

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+language: php
+php:
+  - 5.6
+  - 5.5
+  - 5.4
+  - 5.3
+  - hhvm
+  - nightly
```
Comment (GrahamCampbell) [inline .travis.yml]: :-1: `7.0` makes way more sense.

Comment (GrahamCampbell) [inline .travis.yml]: Instead of nightly...

**Pull request merged**